fybrik/README_mysql.md

```diff
@@ -0,0 +1,185 @@
+
+  ```bash
+ cat << EOF | kubectl apply -f -
+apiVersion: v1
+kind: Secret
+metadata:
+  name: userdata
+type: Opaque
+stringData:
+  username: root
+  password: "${MYSQL_ROOT_PASSWORD}"
+EOF
+  ```
+
+1. Create an asset (the `userdata` asset) and an application that requires this asset:
+   ```bash
+   kubectl apply -f $AIRBYTE_MODULE_DIR/fybrik/read-flow/asset-mysql.yaml -n fybrik-airbyte-sample
+   kubectl apply -f $AIRBYTE_MODULE_DIR/fybrik/read-flow/application.yaml -n fybrik-airbyte-sample
+   ```
+
+1. After the application is created, the Fybrik manager attempts to create the data path for the application. Fybrik realizes that the Airbyte module can give the application access to the `userdata` dataset, and deploys it in the `fybrik-blueprints` namespace. To verify that the Airbyte module was indeed deployed, run:
+   ```bash
+   kubectl get pods -n fybrik-blueprints
+   ```
+
+1. To verify that the Airbyte module gives access to the `userdata` dataset, run:
+   ```bash
+   cd $AIRBYTE_MODULE_DIR/helm/client
+   ./deploy_airbyte_module_client_pod.sh
+   kubectl exec -it my-shell -n default -- python3 /root/client.py --host my-app-read-fybrik-airbyte-sample-airbyte-module.fybrik-blueprints --port 80 --asset fybrik-airbyte-sample/userdata
+   ```
+
+# Writing Dataset with Fybrik Application
+
+In this example, a small dataset is written to mysql table. To do so a FybrikApplication (i.e. the request) must be submitted indicating the desired data set(s) to be written.
+
+As above, you will need a copy of the Fybrik repository (`git clone https://github.com/fybrik/fybrik.git`). Set the following environment variables: FYBRIK_DIR for the path of the `fybrik` directory, and AIRBYTE_MODULE_DIR for the path of the `airbyte-module` directory.
+
+Repeat steps 1-5 above.
+
+6. Setup and initialize mysql for writing a dataset
+
+    1. Deploy [mysql](https://bitnami.com/stack/mysql/helm) helm chart in `fybrik-airbyte-sample` namespace:
+      ```bash
+      helm repo add bitnami https://charts.bitnami.com/bitnami
+      helm install mysql bitnami/mysql -n fybrik-airbyte-sample
+      ```
+    2. Use the instructions from the helm chart notes to run a pod that is use as a client and connect to the service:
+      ```bash
+      MYSQL_ROOT_PASSWORD=$(kubectl get secret --namespace fybrik-airbyte-sample mysql -o jsonpath="{.data.mysql-root-password}" | base64 -d)
+      kubectl run mysql-client --rm --tty -i --restart='Never' --image  docker.io/bitnami/mysql:8.0.32-debian-11-r0 --namespace fybrik-airbyte-sample --env MYSQL_ROOT_PASSWORD=$MYSQL_ROOT_PASSWORD --command -- bash
+      mysql -h mysql.fybrik-airbyte-sample.svc.cluster.local -uroot -p"$MYSQL_ROOT_PASSWORD"
+      ```
+   
+    3. In mysql client shell insert the following commands to upload fake data into a new database called `test`:
```

Comment:
as above
In a mysql client shell prompt

Reply:
Fixed Thanks